app/controllers/proposals_controller.rb

```diff
@@ -23,6 +23,7 @@ def index
   def new
     @proposal = Proposal.new(event: @event)
     @proposal.speakers.build(user: current_user)
+    flash.now[:danger] = @incomplete_profile_msg if @incomplete_profile_msg.present?
```

Comment:
Since we're really giving a warning vs showing a validation error (at this point), we could do this instead:

flash.now[:warning] = incomplete_profile_msg unless current_user.complete?